### ISSUE TITLE
Extractvbs is slow

### ISSUE THREAD

**Issue by francisdb** [vpinball/vpinball#814]
**Describe the enhancement**

The `extractvbs` speed could be improved by not loading the whole table

vpinball: 6.612s total
[vpxtool](https://github.com/francisdb/vpxtool): 0.091s total

**To Reproduce**

vpinball (standalone)

```
> time ../vpinball/VPinballX_GL -extractvbs "Godzilla (Sega 1998) VPW v1.0.vpx"
2023-09-25 09:52:40.171 INFO  [8006818] [VPApp::InitInstance@602] Starting VPX...
2023-09-25 09:52:40.171 INFO  [8006818] [VPApp::InitInstance@605] m_logicalNumberOfProcessors=10
2023-09-25 09:52:40.171 INFO  [8006818] [VPApp::InitInstance@607] m_szMyPath=/Users/francisdb/vpinball/vpinball/
2023-09-25 09:52:40.171 INFO  [8006818] [VPApp::InitInstance@608] m_szMyPrefPath=/Users/francisdb/.vpinball/
2023-09-25 09:52:40.171 INFO  [8006818] [VPApp::InitVPX@742] Loading table from command line option: Godzilla (Sega 1998) VPW v1.0.vpx
2023-09-25 09:52:40.171 INFO  [8006818] [VPinball::ProfileLog@2688] OnInitialUpdate
2023-09-25 09:52:40.171 INFO  [8006818] [InitRegistryOverride@159] Using settings override file from:
2023-09-25 09:52:40.171 INFO  [8006818] [LoadIni@144] Settings file was not found at:
2023-09-25 09:52:40.171 INFO  [8006818] [VPinball::ProfileLog@2688] LoadGameFromFilename Godzilla (Sega 1998) VPW v1.0.vpx
2023-09-25 09:52:40.172 INFO  [8006818] [VPinball::ProfileLog@2688] LoadGameFromStorage
2023-09-25 09:52:40.182 INFO  [8006818] [VPinball::ProfileLog@2688] LoadData
2023-09-25 09:52:41.060 INFO  [8006818] [VPinball::ProfileLog@2688] GameItem
2023-09-25 09:52:41.254 INFO  [8006818] [VPinball::ProfileLog@2688] Sound
2023-09-25 09:52:46.602 INFO  [8006818] [VPinball::ProfileLog@2688] Image
2023-09-25 09:52:46.603 INFO  [8006818] [VPinball::ProfileLog@2688] Font
2023-09-25 09:52:46.618 INFO  [8006818] [VPinball::ProfileLog@2688] Collection
2023-09-25 09:52:46.629 INFO  [8006818] [VPinball::ProfileLog@2688] IEditable PostLoad
2023-09-25 09:52:46.630 INFO  [8006818] [VPinball::ProfileLog@2688] InitTablePostLoad
2023-09-25 09:52:46.630 INFO  [8006818] [VPinball::ProfileLog@2688] UI Post Load Start
2023-09-25 09:52:46.630 INFO  [8006818] [VPinball::ProfileLog@2688] UI Post Load End
2023-09-25 09:52:46.630 INFO  [8006818] [CodeViewer::SaveToFile@1835] filename=Godzilla (Sega 1998) VPW v1.0.vbs
2023-09-25 09:52:46.631 INFO  [8006818] [WinMain@1063] Closing VPX...
../vpinball/VPinballX_GL -extractvbs "Godzilla (Sega 1998) VPW v1.0.vpx"  19.66s user 1.63s system 321% cpu 6.612 total
```

[vpxtool](https://github.com/francisdb/vpxtool)

```
time ../vpxtool extractvbs "Godzilla (Sega 1998) VPW v1.0.vpx"
CREATED Godzilla (Sega 1998) VPW v1.0.vbs
../vpxtool/target/release/vpxtool extractvbs   0.02s user 0.06s system 95% cpu 0.091 total
```

**Comment by EarBiteR** [comment]
Something has changed in the file loading.  I run ziprofs.py which enables me to view and select file from inside a zip.  This was working well on linux using wine.  Now its incredibly slow, almost like a race condition.  From the desktop no problems. But using vp on wine and also the newer linux vpx_gl its unusable.  I wrote a script using zenity and if I extract the table first then is launches immediately as expected and its SO NICE now having a Linux version!

**Comment by francisdb** [comment]
Closing this as users can use vpxtool to export fast (and import).

**Comment by francisdb** [comment]
Looks like this was finally implemented in vpinball: see fbdbbf22cdc97aeed3993d356c22f66c3e80966d
